src/schema.ts

```diff
@@ -719,7 +719,7 @@ export default abstract class Schema<
     if (opts.message === undefined) opts.message = locale.default;
 
     if (typeof opts.test !== 'function')
-      throw new TypeError('`test` is a required parameters');
+      throw new TypeError('The `test` function parameter is a required parameter');
```

Comment:
Why not to mention _parameter_ word once?

> The test function parameter is a required.

Reply:
> The test function parameter is a required.

this is not correct grammar

But in 2a8a5391a409c38763fc93f3de1aa6055dec721e I changed it to this:

```
The `test` function parameter is required
```